examples/heal/remote-nsmgr-death/README.md

```diff
@@ -239,10 +239,10 @@ kubectl exec ${NEW_NSE} -n ${NAMESPACE} -- ping -c 4 172.16.1.103
 
 Restore NSMgr setup:
 ```bash
-kubectl apply -k github.com/networkservicemesh/deployments-k8s/apps/nsmgr?ref=050a5817cb8104b907e2c241a7af38c08d3102d1 -n nsm-system
+kubectl apply -k https://github.com/networkservicemesh/deployments-k8s/apps/nsmgr -n nsm-system?ref=050a5817cb8104b907e2c241a7af38c08d3102d1
```

Comment:
Please, check the script.
"ref=" is not at the end of the line here

Reply:
Thank you, fixed.